src/d3d11/d3d11_texture.cpp

```diff
@@ -175,7 +194,7 @@ namespace dxvk {
     // We must keep LINEAR images in GENERAL layout, but we
     // can choose a better layout for the image based on how
     // it is going to be used by the game.
-    if (imageInfo.tiling == VK_IMAGE_TILING_OPTIMAL && !isMultiPlane)
+    if (imageInfo.tiling == VK_IMAGE_TILING_OPTIMAL && !isMultiPlane && !imageInfo.shared)
```

Comment:
`imageInfo.shared` is used for some internal stuff as well such as swap chain images, where we don't want this, can we have a separate conditional for this?